sormas-backend/src/main/java/de/symeda/sormas/backend/audit/AuditLoggerInterceptor.java

```diff
@@ -101,14 +155,14 @@ public Object logAudit(InvocationContext context) throws Exception {
 		// expression yields true if it IS a local bean => should not be audited and ignored
 		Boolean shouldIgnoreAudit = shouldIgnoreClassCache.computeIfAbsent(target, k -> target.getAnnotationsByType(LocalBean.class).length > 0);
```

Comment:
Your implementation is excluding the AdoServices from being ignored in general, which goes for all their method calls. To circumvent this you have created allowedLocalAuditMethods to only include the delete calls.

Unfortunately you are now also excluding the method calls for all other remote beans, because their methods are not listed in allowedLocalAuditMethods.

What you need to do instead is to by-pass shouldIgnoreAudit if the method is in allowedLocalAuditMethods